README.md

```diff
@@ -54,10 +55,30 @@ sudo apt install python3-dev libpq-dev
 8. Now run `python manage.py runserver`. Go to http://127.0.0.1:8000/ in your browser. You should be able to access our project.
 
 ## Setup Synchronization (Optional)
+
+### SymmetricDS
 _Please make sure there is no whitespace in the path to the cloned repository. SymmetricDS doesn't work with a path with whitespaces._
 
 To setup SymmetricsDs on your machine, click [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/blob/master/symmetricds/README.md) In case you are on windows, please also refer to the wiki on the topic by going [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Setting-up-SymmetricDS-on-Windows) Windows support is not streamlined and is very error prone but this should be enough to get you started. In case you want to help us make this better please refer below as to how you can contribute to the project.
 
+### Heroku Deployment
```

Comment:
can we have a level-2 (##) section for this? because the user may want to use the program only on Heroku, without using it on a local computer.

Reply:
I think it would also be helpful to explain what is Heroku deployment, and what it does, or benefits.

Reply:
Then I this I want to move this section above "Setup Synchronization (Optional)" part, my previous think was that Heroku is also optional since sync part is optional

Reply:
oh sure, my mistake, it makes sense to put it above Sync.

Reply:
> oh sure, my mistake, it makes sense to put it above Sync.

I move entire heroku deployment above sync part

Reply:
awesome!